api/src/test/java/org/apache/iceberg/variants/VariantTestUtil.java

```diff
@@ -107,6 +107,10 @@ private static byte metadataHeader(boolean isSorted, int offsetSize) {
     return (byte) (((offsetSize - 1) << 6) | (isSorted ? 0b10000 : 0) | 0b0001);
   }
 
+  public static VariantValue value(VariantMetadata metadata, ByteBuffer value) {
```

Comment:
Why do we need this helper? Doesn't look like it provides any value over VariantValue.from which has the same signature?